plexupdate.sh

```diff
@@ -21,10 +21,8 @@
 #
 # Returns 0 on success
 #         1 on error
-#         2 if file already downloaded
-#         3 if page layout has changed.
-#         4 if download fails
-#         6 if update was deferred due to usage
+#         2 an update was found/installed
```

Comment:
Add something like `(if NOTIFY=yes, otherwise it will return 0)` to 2 exit code. Removing the detailed exit codes otherwise is fine since user can get much better details through text output.